### PR TITLE
Use assertj fluent style in flowable-engine module.

### PR DESCRIPTION
**Pull request by dbmalkovsky**
This is part 2 for the `org.flowable.examples.bpmn` package. The files converted are after the `gateway` subpackage through and including the `subprocess` package. There is one more part to come.....

### CODE REVIEW

**Comment by dbmalkovsky** [comment]
The JDK 11 job failed because `flowable-mule` could not resolved dependencies.

File: modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java [hunk 1]
```diff
@@ -39,84 +43,63 @@ public void testSetScriptResultToProcessVariable() {
...
-        try {
-            runtimeService.startProcessInstanceByKey("failingScript");
-        } catch (Exception e) {
-            expectedException = e;
-        }
-
-        // Check if correct exception is found in the stacktrace
-        verifyExceptionInStacktrace(expectedException, MissingPropertyException.class);
+        assertThatThrownBy(() -> runtimeService.startProcessInstanceByKey("failingScript"))
+                .hasStackTraceContaining("MissingPropertyException");
```
Comment (filiphr) [inline modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java:54]: Although this is correct. I would prefer to keep the same assertion as before. Verify that the `MissingPropertyException` is in the cause is one of the causes in the stacktrace and not just containing the string

File: modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java [hunk 2]
```diff
@@ -39,84 +43,63 @@ public void testSetScriptResultToProcessVariable() {
...
-        Exception expectedException = null;
-        try {
-            runtimeService.startProcessInstanceByKey("failingScript");
-        } catch (Exception e) {
-            expectedException = e;
-        }
-
-        verifyExceptionInStacktrace(expectedException, IllegalStateException.class);
+        assertThatThrownBy(() -> runtimeService.startProcessInstanceByKey("failingScript"))
+                .hasStackTraceContaining("IllegalStateException");
```
Comment (filiphr) [inline modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java:61]: Some here

File: modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/servicetask/JavaServiceTaskTest.java [hunk 3]
```diff
@@ -132,36 +135,29 @@ public void testExpressionFieldInjectionWithSkipExpression() {
...
         Execution execution2 = runtimeService.createExecutionQuery().processInstanceId(pi2.getId()).activityId("waitState").singleResult();
 
-        assertNotNull(execution2);
+        assertThat(execution2).isNotNull();
 
         Map<String, Object> pi2VarMap = runtimeService.getVariables(pi2.getProcessInstanceId());
-        assertFalse(pi2VarMap.containsKey("var1"));
-        assertFalse(pi2VarMap.containsKey("var2"));
+        assertThat(pi2VarMap)
+                .doesNotContainKey("var1")
```
Comment (filiphr) [inline modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/servicetask/JavaServiceTaskTest.java:142]: Shouldn't this be `containsKey`?

File: modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java (hunk 2 above)
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java]: @filiphr and fixed here.

File: modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java (hunk 1 above)
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java]: @filiphr OK; fixed.

File: modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/servicetask/JavaServiceTaskTest.java (hunk 3 above)
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/servicetask/JavaServiceTaskTest.java:142]: @filiphr No, note that the original assertion was `assertFalse`.

Comment (filiphr) [reply modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/servicetask/JavaServiceTaskTest.java:142]: Yes you are 100% right. I only read the `containsKey`. My mistake sorry.

File: modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java (hunk 1 above)
Comment (filiphr) [reply modules/flowable-engine/src/test/java/org/flowable/examples/bpmn/scripttask/ScriptTaskTest.java]: Thanks for both fixes

**Comment by dbmalkovsky** [comment]
The JDK11 build fails with timing issues.

**Pull request merged**